feat: new lacework_alert_channel_webhook resource (#49)
lacework/terraform-provider-lacework: website/docs/r/alert_channel_webhook.html.markdown

---
subcategory: "Alert Channels"
layout: "lacework"
page_title: "Lacework: lacework_alert_channel_webhook"
description: |-
  Create and manage Webhook Alert Channel integrations
---

# lacework\_alert\_channel\_webhook

> Configure Lacework to forward alerts to a 3rd party Webhook.

Reviewer: ```suggestion
You can use this resource to create a custom webhook that receives Lacework alert notifications from a Lacework alert channel and forwards those alerts to a 3rd party application.
```

> 

Reviewer: here

> -> **Note:**  If the third-party that is receiving the HTTP POST request requires an API token, enter the API Token as part of the URL eg. https://webhook.com?api-token=123

Reviewer: ```suggestion
-> **Note:**  If the third-party application receiving the HTTP POST request requires an API token, enter the API token as part of the URL eg. https://webhook.com?api-token=123
```

---
subcategory: "Alert Channels"
layout: "lacework"
page_title: "Lacework: lacework_alert_channel_webhook"
description: |-
  Create and manage Webhook Alert Channel integrations
---

# lacework\_alert\_channel\_webhook

You can use this resource to create a custom webhook that receives Lacework alert notifications from a Lacework alert channel and forwards those alerts to a third-party application.

-> **Note:**  If the third-party application receiving the HTTP POST request requires an API token, enter the API token as part of the URL eg. https://webhook.com?api-token=123

> To find more information about the payload view the lacework support documentation

Reviewer: Lacework needs to be written capitalized.

Author: Ah, missed that. Thanks!

> To find more information about the payload view the lacework support documentation
> https://support.lacework.com/hc/en-us/articles/360034367393-Webhook

Reviewer: ```suggestion
To find more information about the alert payload, see the [Lacework support documentation](https://support.lacework.com/hc/en-us/articles/360034367393-Webhook).
```
